app/assets/javascripts/discourse/app/templates/navigation/category.hbs

```diff
@@ -1,9 +1,8 @@
 <AddCategoryTagClasses @category={{this.category}} />
 
 <section class="category-heading">
-  {{#if this.category.uploaded_logo.url}}
-    <CdnImg @src={{this.category.uploaded_logo.url}} @class="category-logo" @width={{this.category.uploaded_logo.width}} @height={{this.category.uploaded_logo.height}} @alt="" />
-
+  {{#if (or this.category.uploaded_logo.url this.category.uploaded_logo_dark.url)}}
```

Comment:
```suggestion
  {{#if this.category.uploaded_logo.url}}
```

I think we should keep a simpler rule here (and elsewhere): 

- only category logo is required for the display of the `CategoryLogo` component
- the dark logo should be only displayed as an alternative, that is, if the regular `category.uploaded_logo` is empty, then dark logo is not included in the HTML